SoftLayer/CLI/routes.py

```diff
@@ -163,7 +164,6 @@
     ('image:list', 'SoftLayer.CLI.image.list:cli'),
     ('image:import', 'SoftLayer.CLI.image.import:cli'),
     ('image:export', 'SoftLayer.CLI.image.export:cli'),
-    ('image:datacenter', 'SoftLayer.CLI.image.datacenter:cli'),
```

Comment:
Looks like you might have accidentally removed this line. Or your branch isn't quite up to date with master. Either way please add this line back.